From: Guido Guenther <agx@sigxcpu.org>
Date: Sat, 14 Jun 2008 18:53:15 +0200
Subject: [PATCH] set a more gnomeisch invisible-char

---
 src/krb5-auth-dialog.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/src/krb5-auth-dialog.c b/src/krb5-auth-dialog.c
index c443cd3..6b48d2b 100644
--- a/src/krb5-auth-dialog.c
+++ b/src/krb5-auth-dialog.c
@@ -924,6 +924,7 @@ ka_create_gtk_secure_entry (GladeXML *xml G_GNUC_UNUSED,
 	if (!strcmp(name, "krb5_entry")) {
 		entry = gtk_secure_entry_new ();
 		gtk_secure_entry_set_activates_default(GTK_SECURE_ENTRY(entry), TRUE);
+		gtk_widget_set(entry, "invisible-char", 0x25cf, NULL);
 		gtk_widget_show (entry);
 	} else {
 		g_warning("Don't know anything about widget %s", name);
-- 
